src/python/pants/backend/go/util_rules/first_party_pkg_test.py

```diff
@@ -16,6 +16,7 @@
     first_party_pkg,
     go_mod,
     link,
+    pkg_analyzer,
```

Comment:
FYI you could avoid this boilerplate by registering `pgk_analyzer.rules()` in the `rules()` functions for `first_party_pkg.py` and `third_party_pkg.py`. We use that pattern in Python backend. The engine de-duplicates the rules.

Reply:
Done.